CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ A more detailed list of changes is available in the corresponding milestones for
 ### Note-worthy code changes
   - Renamed `multiprocessing.py` to `multiproc.py` to avoid conflict with Python
     stdlib module of the same name in some configurations.
+  - Re-worked `cff.py` checks using `@condition` to avoid repeated iterations
+    over the glyph set.
+  - Added checks for deprecated CFF operators `dotsection` and `seac` including
+    "implied" `seac` that uses extra arguments to `endchar`.
 
 ## 0.7.30 (2020-Aug-??)
 ### Note-worthy code changes
```

Comment:
Please also merge the two entries for 0.7.31 and 0.7.30 as we have not yet released 0.7.30. I am planning to release it very soon this week, so it will be `2020-Sept-someday`

Reply:
Done in https://github.com/googlefonts/fontbakery/pull/3033/commits/66b8ecf818c9ec236bdb0fdf75a3bcde982f8b51